tools/bin/ci_performance_test.sh

```diff
@@ -46,14 +44,24 @@ else
   elif [[ "$connector" == *"connectors"* ]]; then
     connector_name=$(echo $connector | cut -d / -f 2)
     selected_performance_test=$(echo "$all_performance_tests" | grep "^$connector_name$" || echo "")
-    performanceTestCommand="$(_to_gradle_path "airbyte-integrations/$connector" performanceTest) $firstarg $secondargt"
+    performanceTestCommand="$(_to_gradle_path "airbyte-integrations/$connector" performanceTest)"
   else
     selected_performance_test=$(echo "$all_performance_tests" | grep "^$connector$" || echo "")
-    performanceTestCommand=":airbyte-integrations:connectors:$connector:performanceTest $firstarg $secondarg"
+    performanceTestCommand=":airbyte-integrations:connectors:$connector:performanceTest"
   fi
   if [ -n "$selected_performance_test" ] ; then
-    echo "Running: ./gradlew --no-daemon --scan $performanceTestCommand"
-    ./gradlew --no-daemon --scan "$performanceTestCommand"
+    if [[ "$firstarg" ]]; then
+      if [[ "$secondarg" ]]; then
+        echo "Running: ./gradlew --no-daemon --scan $performanceTestCommand $firstarg $secondarg"
+        ./gradlew --no-daemon --scan "$performanceTestCommand" "$firstarg" "$secondarg"
+      else
+        echo "Running: ./gradlew --no-daemon --scan $performanceTestCommand $firstarg"
+        ./gradlew --no-daemon --scan "$performanceTestCommand" "$firstarg"
+      fi
+    else
+      echo "Running: ./gradlew --no-daemon --scan $performanceTestCommand"
+      ./gradlew --no-daemon --scan "$performanceTestCommand"
+    fi
```

Comment:
Instead of checking if there is one or two parameters, you can keep an `allArgs` variable that concatenate both `firstarg` and `secondarg` together. If either of the arguments is empty or null, they would just be some extra spaces, which is not a big deal. Something like:

```bash
allArgs=" ${firstarg} ${secondarg}"
echo "Running: ./gradlew --no-daemon --scan $performanceTestCommand $allArgs"
./gradlew --no-daemon --scan "$performanceTestCommand" "$allArgs"
```

This can eliminate all the `if` checks.

Reply:
I was thinking about this entry - allArgs = "$ {firstarg} $ {secondarg}". Unfortunately, the gradle will not treat such a record as two arguments, but as one.